packages/notification/docs.mdx

```diff
@@ -75,3 +75,22 @@ Notification component to be used within our [Toaster](/components/toast) compon
   onDismiss={() => {}}
 />
 `}</LiveComponent>
+
+## With actions
+
+<LiveComponent>{`
+<NotificationWithActions
+  heading="Notification heading"
```

Comment:
🙋 Do we want to add `onDismiss={() => {}}` in here to show that this prop is also available in the `NotificationWithActions` component?

Reply:
Yeah, thats a good callout. Added https://github.com/hashicorp/react-components/pull/593/commits/2e302518dfe231a3050c33cea08800ab060dd5db